app/src/main/java/org/simple/clinic/di/DateFormatterModule.kt

```diff
@@ -18,9 +20,12 @@ import javax.inject.Named
 class DateFormatterModule {
 
   @Provides
-  fun providesChronology(country: Country): Chronology {
+  fun providesChronology(country: Country, features: Features): Chronology {
+    if (!features.isEnabled(Feature.EthiopianCalendar))
+      return IsoChronology.INSTANCE
+
     return when (country.isoCountryCode) {
-      Country.ETHIOPIA -> EthiopicChronology.INSTANCE
+      Country.ETHIOPIA -> return EthiopicChronology.INSTANCE
       else -> IsoChronology.INSTANCE
     }
```

Comment:
Can we try and clean this up so that there's only a single `return` statement? Seems like there are too many at the moment.